models.py

```diff
@@ -61,3 +61,9 @@ def __iter__(self):
         iter_keys = ["id","datetimestamp","bin_id"]
         iter_vals = [str(self.id), str(self.datetimestamp), str(self.bin_id)]
         return iter(zip(iter_keys, iter_vals))
+
+class BarcodeItem(db.Model):
+    __tablename__ = "barcodes"
+    barcode = db.Column(db.Integer, primary_key=True)
```

Comment:
The barcode field should be a string because even though most barcodes have numbers in them, it is possible for barcodes to have leading zeroes like 0001223123 or even have some letters in them